clients/lcdproc/disk.c

```diff
@@ -124,32 +137,71 @@ disk_screen(int rep, int display, int *flags_ptr)
 	sock_printf(sock, "widget_set D f 1 2 %i %i %i %i v 12\n", lcd_wid, lcd_hgt, lcd_wid, count);
 	for (i = 0; i < count; i++) {
 		char tmp[lcd_wid + 1];	/* should be large enough */
+		char cap[8] = {'\0'};
+		char dev[256];
+		u_int64_t size = 0;
+		int full = 0;
 
-		if (table[i].dev[0] == '\0')
+		// Eewww, goto... but shy of restructuring everything it's 
+		// the cleanest way to skip the rest of the outer loop. :-/
+		next_mnt:
+
```

Comment:
I'm kind of a fan of goto and use it regularly. But jumping backwards in the program always is more confusing then other uses.

Reply:
Resolved in #182